### ISSUE TITLE
Pool vs algorithm heirarchy

### ISSUE THREAD

**Issue by Kelarik** [aaronsace/MultiPoolMiner#453]
I noticed an update in a recent commit about -algorithm superceeding -pool, something I'd noticed myself when just messing around and throwing in every single supported algorithm. Is it possible to do it the other way around? For example, having a setup where you can go "-poolname Zpool,Nicehash -algorithm All", and have the miner restrict itself to just the algorithms that are supported on those pools

**Comment by aaronsace** [comment]
No, you should restrict the algorithms; most algorithms are useless. 

The latest version will rarely switch pools; it's about the algorithms.

**Comment by fonyo** [comment]
As described in #433 and in the -algorithm section of the README, this is an intended feature to overcome zpool not having ethash at all. This is considered to be more important due to ethash being one of the most profitable algos.
